.kitchen.yml

```diff
@@ -86,3 +86,21 @@ suites:
         listen_http: 8080
     java:
       jdk_version: '8'
+- name: kibana4_package
```

Comment:
Should we name this more explicitly to note that it is apache - like kibana4_apache_package?